lib/cn/vblock_reader.c

```diff
@@ -98,10 +98,10 @@ vbr_desc_update(
 void
 vbr_readahead(
     struct vblock_desc *     vbd,
-    u32                      voff,
-    u32                      vlen,
+    off_t                    voff,
+    size_t                   vlen,
     u32                      ra_flags,
-    u32                      ra_len,
+    size_t                   ra_len,
```

Comment:
voff is now signed, i hope there isn't any issue with that...

Reply:
reading the code, shit would hit the fan if voff was signed

Reply:
Let me just revert that